.pylintrc

```diff
@@ -0,0 +1,37 @@
+[MAIN]
+
+# Specify a score threshold under which the program will exit with error.
+fail-under=10
+
+[BASIC]
+
+# Minimum line length for functions/classes that require docstrings, shorter
+# ones are exempt.
+docstring-min-length=100
+
+
+# Good variable names which should always be accepted, separated by a comma.
+good-names=s, f, i, j, k, e, _
+
+# Include a hint for the correct naming format with invalid-name.
+include-naming-hint=yes
+
+[FORMAT]
+
+# Maximum number of characters on a single line.
+max-line-length=120
```

Comment:
As above.

Reply:
Yes this one should agree with the other (which is currently 99 - Python was traditionally 79, but 99 is more practical now that typing is creeping in everywhere)